libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -360,6 +360,9 @@ instance (Arbitrary cfg, IsFeatureConfig cfg) => Arbitrary (WithStatusPatch cfg)
 -- if we switch to `unlocked`, we auto-enable the feature, and if we switch to locked, we
 -- auto-disable it.  But we need to change the API to force clients to use `lockStatus`
 -- instead of `status`, current behavior is just wrong.
+--
+-- FUTUREWORK(paolo): why delete this type? it's used as the request body of
```

Comment:
fair.  i don't know how deletion follows from my rant about lock status.  feel free to remove both FUTUREWORKs, or better yet move the rant to somewhere suitable, eg, the lock status type.

Reply:
I added a note explaining this lock/disable business.